core/src/OpenMPTarget/Kokkos_OpenMPTarget_ParallelFor_Team.hpp

```diff
@@ -141,11 +141,6 @@ class ParallelFor<FunctorType, Kokkos::TeamPolicy<Properties...>,
 // the clang compiler. atomic_compare_exchange can be avoided since the standard
 // guarantees that the number of teams specified in the `num_teams` clause is
 // always less than or equal to the maximum concurrently running teams.
-#if !defined(KOKKOS_IMPL_OPENMPTARGET_HIERARCHICAL_INTEL_GPU)
-    KOKKOS_IMPL_OMPTARGET_PRAGMA(
-        teams thread_limit(team_size) firstprivate(a_functor)
-            num_teams(max_active_teams) is_device_ptr(scratch_ptr)
-                KOKKOS_IMPL_OMPX_DYN_CGROUP_MEM(shmem_size_L0))
```

Comment:
Why does this go away. It was the non-Intel path already?

Reply:
Oh yeah that was a mistake, I should not have deleted this macro. I fixed it.